api-authorization/server/authorization-middleware.js

```diff
@@ -0,0 +1,17 @@
+const jwt = require('jsonwebtoken'); // eslint-disable-line
+const ClientError = require('./client-error'); // eslint-disable-line
+
+function authorizationMiddleware(req, res, next) {
+  /* your code here */
+  const token = req.get('x-access-token');
+  if (!token) {
+    throw new ClientError(401, 'authentication required');
+  } else {
+    const jwtVerify = jwt.verify(token);
```

Comment:
`jwt.verify` takes 2 arguments, the first argument is the `token` the second is the `TOKEN_SECRET`
```JAVASCRIPT
const payload = jwt.verify(token, process.env.TOKEN_SECRET);
```